client/src/components/NewAttractionForm.js

```diff
@@ -71,15 +81,33 @@ const NewAttractionForm = (props) => {
 
   const handleSubmit = (event) => {
     event.preventDefault();
+    const body = new FormData();
+    body.append("image", newAttraction.image);
+    props.addNewAttraction(body);
```

Comment:
I'm a little confused why there's both `props.addNewAttraction(body)` and a call on `postAttraction`? It also looks like `FormData` is created twice - above in this `handleSubmit`, and in the `postAttraction`. I don't see where the function `addNewAttraction` comes from?